src/sites/twitch-twilight/modules/chat/input.jsx

```diff
@@ -324,10 +324,18 @@ export default class Input extends Module {
 		inst.tempInput = '';
 		inst.messageHistoryPos = -1;
 
+		let element = this.fine.getChildNode(inst);
+		let container = element && element.querySelector('div[class="Layout-sc-nxg1ff-0 chat-input__input-icons"]');
```

Comment:
This selector is very prone to break. When at all possible, you should avoid using CSS class names from `styled-components` as they are generated by Twitch's build process and subject to change every time Twitch releases an update.

Using `chat-input__input-icons` should be plenty.